src/components/BatchesPage/BatchesPage.js

```diff
@@ -135,4 +166,90 @@ const ActionHeader = () => (
   </span>
 );
 
+const BatchesModal = (props) => {
+  const {
+    isOpen,
+    toggleModalFunction,
+    className,
+    batchDetails,
+  } = props;
+  return (
+    <Modal isOpen={isOpen} toggle={toggleModalFunction} className={className}>
+      <ModalHeader toggle={toggleModalFunction}>
+        <Row>
+          <Col>
+            Batch Information
+          </Col>
+        </Row>
+      </ModalHeader>
+      <ModalBody>
+        <Row>
+          <Col>
+            Batch Name
+            &nbsp;-&nbsp;
+            {batchDetails.batchID}
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+            status
+            &nbsp;-&nbsp;
+            {batchDetails.status === 0 ? 'Inactive' : 'Active' }
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+            students
+            &nbsp;-&nbsp;
+            {batchDetails.students}
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+            creationDate
+            &nbsp;-&nbsp;
```

Comment:
`&nbsp` is not really used now. If you can do it using alternate ways, that's great!

Reply:
changed 👍🏻
self approving